tuf/data/roles.go

```diff
@@ -214,6 +214,18 @@ type Role struct {
 	Paths []string `json:"paths,omitempty"`
 }
 
+func isValidRootRoleStructure(r RootRole, validKeys Keys) error {
```

Comment:
I was thinking about that, but I wasn't sure which way we decided - is there ever a use case for removing all keys and invalidating the role without deleting it?